apps/site/src/pages/[shortname]/types/entity-type/[...slug-maybe-version].page/entity-type-edit-bar.tsx

```diff
@@ -0,0 +1,79 @@
+import {
```

Comment:
I think we should probably hide the `Coming Soon` banner in this screen. And maybe use a similar styling for it?

![image](https://user-images.githubusercontent.com/37453800/220107896-24821dbf-6ce9-4ecc-9a9e-9d3498cd5003.png)

Reply:
Actually will the `Coming Soon` banner stick around after the 28th?

Reply:
The `Coming Soon` banner will be removed when we merge the `0.3` branch so this will solve itself

Reply:
Well, we still need to introduce the code that does that – I'll remove in follow up